instrumentation/testing/testing.go

```diff
@@ -154,6 +161,36 @@ func (test *Test) End() {
 	test.span.Finish()
 }
 
+func (test *Test) extractTestLoggerOutput() {
+	output := extractTestOutput(test.t)
+	if output == nil {
+		return
+	}
+	outStr := string(*output)
+	for _, matches := range TESTING_LOG_REGEX.FindAllStringSubmatch(outStr, -1) {
+		file := matches[1]
+		line := matches[2]
+		message := strings.ReplaceAll(matches[3], "\n        ", "\n")
```

Comment:
Does this work with subtests?

Reply:
yes, I've just tested

Reply:
Added a new test in the new commit to run the regex inside a subtest